project/src/js/ui/tournament/detail/tournamentDetailView.js

```diff
@@ -94,6 +139,21 @@ function variantInfo(data) {
   return variant;
 }
 
+function variantKey(data) {
```

Comment:
You should not have to deduce the variant key from the time. Actually this info you get it from the server. And it's not `variant` key but `perf` key which have the `blitz` or `classical`, etc values.